checks/libs/rccl/check_rccl-tests-sing.py

```diff
@@ -0,0 +1,241 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+from reframe.core.backends import getlauncher
+
+
+class fetch_rccl(rfm.RunOnlyRegressionTest):
+    '''Fixture for fetching RCCL.'''
+    local = True
+    url_tag = 'rocm-5.2.3.tar.gz'
+    file_name = f'rccl-{url_tag}'
+    executable = f'curl -LJO https://github.com/ROCmSoftwarePlatform/rccl/archive/refs/tags/{url_tag}'  # noqa: E501
+    postrun_cmds = [
+        f'tar xzf {file_name}'
+    ]
+
+    @sanity_function
+    def validate_download(self):
+        return sn.assert_eq(self.job.exitcode, 0)
+
+
+class fetch_rccl_tests(rfm.RunOnlyRegressionTest):
+    '''Fixture for fetching the rccl-tests.'''
+    local = True
+    repo_name = 'rccl-tests'
+    executable = f'git clone -b develop git@github.com:ROCmSoftwarePlatform/{repo_name}.git'  # noqa: E501
+    postrun_cmds = [
+        f'cd {repo_name};'
+        'git checkout 3fbd328'
+    ]
+
+    @sanity_function
+    def validate_download(self):
+        return sn.assert_eq(self.job.exitcode, 0)
+
+
+class fetch_aws_ofi_rccl_plugin(rfm.RunOnlyRegressionTest):
+    '''Fixture for fetching the AWS libfabric plugin.'''
+    local = True
+    repo_name = 'aws-ofi-rccl'
+    executable = f'git clone -b cxi git@github.com:ROCmSoftwarePlatform/{repo_name}.git'  # noqa: E501
```

Comment:
The same here:
git clone -b develop https://github.com/ROCmSoftwarePlatform/{repo_name}.git